docs/accessibilityinfo.md

```diff
@@ -21,58 +21,57 @@ Sometimes it's useful to know whether or not the device has a screen reader that
 <block class="functional syntax" />
 
 ```SnackPlayer name=AccessibilityInfo%20Function%20Component%20Example
-
-import React, { useState, useEffect } from "react";
-import { AccessibilityInfo, View, Text, StyleSheet } from "react-native";
+import React, {useState, useEffect} from 'react';
```

Comment:
So long as I'm think of this I'm not sure if disabling [bracket spacing](https://prettier.io/docs/en/options.html#bracket-spacing) is a good change. I think that having those spaces improves overall readability of the code (especially when function uses object as a parameter).